logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/reaction/ReactionRepository.kt

```diff
@@ -0,0 +1,103 @@
+package com.wire.kalium.logic.data.message.reaction
+
+import com.wire.kalium.logic.StorageFailure
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.id.IdMapper
+import com.wire.kalium.logic.data.message.UserReactions
+import com.wire.kalium.logic.data.user.UserId
+import com.wire.kalium.logic.di.MapperProvider
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.wrapStorageRequest
+import com.wire.kalium.persistence.dao.reaction.ReactionDAO
+
+interface ReactionRepository {
+    suspend fun persistReaction(
+        originalMessageId: String,
+        conversationId: ConversationId,
+        senderUserId: UserId,
+        date: String,
```

Comment:
Why not use an `Instant` and convert it to `String` or something else in the persistence layer.

Reply:
Definitly something we need all across the app when it comes to persistence module.

It's one of my pending tasks to be done Soon™.
Maybe I'll tackle this during this week.

Reply:
Ok, so this will be tackled separately.